delhost.php

```diff
@@ -1,25 +0,0 @@
-#!/usr/bin/env php
```

Comment:
Looks good overall, however can we just print a warning when using delhost.php that it's now deprecated and will be removed in a future version. I expect people have done some form of automation surround it so we shouldn't just drop it.

Reply:
done